tbans/requests/webhook_request.py

```diff
@@ -72,9 +74,16 @@ def _json_string(self):
         import json
         return json.dumps(json_dict, ensure_ascii=True)
 
+    # This checksum is insecure and has been deprecated in favor of an HMAC
     def _generate_webhook_checksum(self, payload):
         import hashlib
         ch = hashlib.sha1()
         ch.update(self.secret)
         ch.update(payload)
         return ch.hexdigest()
+
+    def _generate_webhook_hmac(self, payload):
+        import hashlib
+        import hmac
+
+        return hmac.new(str(self.secret), str(payload), hashlib.sha256).hexdigest()
```

Comment:
We shouldn't *need* to `str` payload, because we use `ensure_ascii=True` which ensures we'll get a `str` back with only ASCII characters. But this feels like an implementation detail, so it's probably fine